src/features/indexer/indexerSlice.ts

```diff
@@ -86,12 +70,24 @@ export const indexerSlice = createSlice({
   extraReducers: (builder) => {
     builder.addCase(fetchIndexerUrls.fulfilled, (state, action) => {
       state.indexerUrls = action.payload;
+      if (!action.payload.length) {
+        state.errorText = i18n.t("orders.noIndexersFound");
+      }
     });
     builder.addCase(getFilteredOrders.fulfilled, (state, action) => {
       state.orders = action.payload;
+      if (!action.payload.length) {
+        state.errorText = i18n.t("orders.noIndexerOrdersFound");
```

Comment:
Length 0 is not an error. It's a valid result.

Reply:
You should add a `isLoading`, `noIndexersFound` booleans to indexer store.

```
builder.addCase(getFilteredOrders.pending, (state, action) => {
      state.isLoading = true;
    })
```

Then do all translations outside in components, not in the store.

Then in `AvailableOrdersList` line 128:

If orders.length === 0 and isLoading === false then `i18n.t("orders.noIndexerOrdersFound");`.